### ISSUE TITLE
correct command line usage

### ISSUE THREAD

**Issue by mikebenfield** [imbs-hl/ranger#341]
I'm using Ranger from the command line. I'm getting runtime performance and accuracy scores that are much worse than, for instance, scikit-learn, so I'm wondering if I'm using it wrong. 

I'm trying to use the extremely randomized trees for classification.

Here's how I'm training Ranger:
```
./ranger --verbose --treetype 1 --ntree 200 --targetpartitionsize 2 --randomsplits 8 --nthreads 3  --file ~/Data/CovType/data_ranger.train --depvarname output --outprefix ~/Data/CovType/rangerout --skipoob --splitrule 5 --write --mtry 8
```

And how I'm making predictions:
```
./ranger --verbose --predict ~/Data/CovType/rangerout.forest --file ~/Data/CovType/data_ranger.test --outprefix ~/Data/CovType/rangerout2
```

Training takes around 7 minutes (as opposed to 2-3 minutes for scikit-learn) and accuracy is something around 80% (as opposed to 96% for scikit-learn). 

The first couple rows of my training data look like this:
```
a0 a1 a2 a3 a4 a5 a6 a7 a8 a9 a10 a11 a12 a13 a14 a15 a16 a17 a18 a19 a20 a21 a22 a23 a24 a25 a26 a27 a28 a29 a30 a31 a32 a33 a34 a35 a36 a37 a38 a39 a40 a41 a42 a43 a44 a45 a46 a47 a48 a49 a50 a51 a52 a53 output
3080.000000 48.000000 19.000000 85.000000 23.000000 2985.000000 224.000000 196.000000 99.000000 5778.000000 1.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 1.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0
```

And the first couple rows of the data I'm predicting for looks like this:
```
a0 a1 a2 a3 a4 a5 a6 a7 a8 a9 a10 a11 a12 a13 a14 a15 a16 a17 a18 a19 a20 a21 a22 a23 a24 a25 a26 a27 a28 a29 a30 a31 a32 a33 a34 a35 a36 a37 a38 a39 a40 a41 a42 a43 a44 a45 a46 a47 a48 a49 a50 a51 a52 output
2582.000000 144.000000 10.000000 420.000000 54.000000 711.000000 235.000000 238.000000 133.000000 1316.000000 0.000000 0.000000 1.000000 0.000000 0.000000 1.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000
```

Is this the right format for the data and the correct command line invocation? Thanks for any help.

**Comment by mnwright** [comment]
Are you sure you want to do classification? Your test data doesn't look like it. Could you share the data somehow? Or simulate something similar?

**Comment by mikebenfield** [comment]
Yes, in the training data, the last column (here with a value of 0) is supposed to be the class. The data is just the forest covertype data here:
https://archive.ics.uci.edu/ml/datasets/covertype

(although I changed the classes to be 0-6 rather than 1-7).

**Comment by mikebenfield** [comment]
Oh, I see you say the _test_ data doesn't look like it. How should the test data be different?

**Comment by mnwright** [comment]
I was just wondering about the `0.000000` in the test data, but that shouldn't be the problem. 
Could you please provide a reproducible example? If possible, please also include the scikit-learn run.

**Comment by mikebenfield** [comment]
Great; thanks for the help!

**Comment by mnwright** [comment]
Please reopen if needed
